frontend/src/lib/k8s/api/v2/hooks.ts

```diff
@@ -55,6 +55,14 @@ export interface QueryResponse<DataType, ErrorType> {
 export interface QueryListResponse<DataType, ItemType, ErrorType>
   extends QueryResponse<DataType, ErrorType> {
   items: Array<ItemType> | null;
+  /**
+   * Results from individual clusters. Keyed by cluster name.
+   */
+  clusterResults?: Record<string, QueryListResponse<DataType, ItemType, ErrorType>>;
+  /**
+   * Errors from individual clusters. Keyed by cluster name.
+   */
+  clusterErrors?: Record<string, ApiError | null> | null;
```

Comment:
Maybe we could extend the previous interface, rather than adding more members to this one? Since it's a new function, having a new interface doesn't seem inadequate.

Reply:
I removed useKubeObjectLists, and now just useKubeObjectList is exported. The multi cluster version is internal.